packages/react-lowcode/src/localization/json.ts

```diff
@@ -27,16 +32,33 @@ export function parseLocaleAST(ast: SourceFile, languageLocale = "en") {
 }
 
 // it preserves original JSON formatting
-export function patchLocaleJSON(localeFile: string, changedMessages: Message[], originalMessages: Message[]) {
-  for (let i = changedMessages.length; i >= 0; i--) {
-    if (changedMessages[i]?.value == originalMessages[i]?.value) {
-      console.log("Equal")
-    } else {
-      const before = localeFile.substring(0, originalMessages[i].position.pos + 1)
-      const after = localeFile.substring(originalMessages[i].position.end - 1)
-      localeFile = before + changedMessages[i].value + after
+export function patchLocaleJSON( 
```

Comment:
I am thinking:
- you can rename this function to `putLocaleMessage()` and keep the original function `patchLocaleJSON` for backward compatibility
- we have duplicit parameters in the function `originalMessages` and `localeFile` - what if we calculate `originalMessages` if not set (we could allow not to pass this duplicit argument)